### PR TITLE
Lisna Anya

### PR DESCRIPTION
**Pull request by AnyaLisna**

### CODE REVIEW

File: src/script.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+       }
+    }
+
+    getItem(id) {
+        return this.todo.find((item)=>item.id === id) || null;
+    }
+
+    next() {
+       this.todo.sort((a, b)=> b.priority - a.priority);
+       return this.todo[0];
```
Comment (gestap) [inline src/script.js]: base on requirements `if there are no items in the list, throw an error`

File: src/script.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+       this.id++;
+       this.todo.push({id: this.id, title: `${title}`, priority: priority});
+       return this.id;
+    }
+
+    removeItem(id) {
+       const index = this.todo.findIndex((item)=>item.id===id);
+       if(index < 0) {
+           return false;
+       } else {
```
Comment (gestap) [inline src/script.js]: try to change checking in `if` and remove this `else`

File: src/script.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+class TODO {
+    constructor() {
```
Comment (gestap) [inline src/script.js]: use 2 whitespaces instead of 4

File: src/script.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    }
+
+    addItem(title, priority) {
+       this.id++;
+       this.todo.push({id: this.id, title: `${title}`, priority: priority});
+       return this.id;
+    }
+
+    removeItem(id) {
+       const index = this.todo.findIndex((item)=>item.id===id);
```
Comment (gestap) [inline src/script.js]:
```suggestion
       const index = this.todo.findIndex(item => item.id === id);
```

File: src/script.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+       if(index < 0) {
+           return false;
+       } else {
+           this.todo.splice(index, 1);
+           return true;
+       }
+    }
+
+    getItem(id) {
+        return this.todo.find((item)=>item.id === id) || null;
```
Comment (gestap) [inline src/script.js]:
```suggestion
        return this.todo.find(item => item.id === id) || null;
```

File: src/script.js [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+    }
+
+    next() {
+       this.todo.sort((a, b)=> b.priority - a.priority);
```
Comment (gestap) [inline src/script.js]:
```suggestion
       this.todo.sort((a, b) => b.priority - a.priority);
```

**Review by ost-ptk — CHANGES_REQUESTED** [review]

File: src/script.js [hunk 7]
```diff
@@ -1,33 +1,35 @@
...
-       return this.todo[0];
+  next() {
+    if(this.todo.length) {
+      this.todo.sort((a, b) => b.priority - a.priority);
+      return this.todo[0];
     }
-}
+    throw new Error('There are no items')
+  }
+}
```
Comment (ost-ptk) [inline src/script.js:35]: the document must end with one empty line

**Review by ost-ptk — APPROVED** [review]

**Pull request closed without merging**